ciao-controller/main.go

```diff
@@ -136,6 +136,8 @@ func main() {
 	identityURL = clusterConfig.Configure.IdentityService.URL
 	serviceUser = clusterConfig.Configure.Controller.IdentityUser
 	servicePassword = clusterConfig.Configure.Controller.IdentityPassword
+	keyringPath = clusterConfig.Configure.Storage.SecretPath
```

Comment:
Okay and then in the case of the ciao-launcher I'll do the same with replacing the hardcoded value with a default value.